src/coreclr/pal/src/thread/context.cpp

```diff
@@ -169,6 +171,30 @@ typedef int __ptrace_request;
 	ASSIGN_REG(X27)     \
 	ASSIGN_REG(X28)
 
+#elif defined(HOST_S390X)
+#define ASSIGN_CONTROL_REGS \
+        ASSIGN_REG(PSWMask) \
```

Comment:
A nit - can you please indent these as the ones for other platforms?

Reply:
Hmm, looks like a space vs. tabs mismatch.   I'm using 8 spaces, like **most** of the other platforms, with the single exception of ARM64 (the immediately preceding one), which uses tabs.   Would you like me to stay with the 8 spaces or switch to tabs?

Reply:
I wanted to say that we don't use tabs in this repo, only spaces. But it seems this file is wrong, using tabs. 
Our setting is 4 spaces per tab.

Reply:
Then it looks like my use of 8 spaces is correct here (matches Intel etc.), right?   The ARM64 definitions use a single tab, they should probably be switched to 8 spaces then as well to match the rest.

Reply:
Yes, you are right.